src/main/java/com/google/sps/servlets/MockNewsService.java

```diff
@@ -3,13 +3,30 @@
 import java.util.List;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.time.Instant;
 import java.lang.IllegalArgumentException;
 
 class MockNewsService implements NewsService {
   @Override
   public List<Article> getWorldNews(int count) {
-    return new ArrayList<>();
+    String[] cities = {"Mountain View", "Los Angeles", "Chicago", "New York", "Austin", "Boston"};
+    int[] counts = {6, 2, 1, 10, 3, 7};
+    List<Article> articles = new ArrayList<>();
+    for (int i = 0; i < cities.length; i++) {
+        for (int j = 0; j < counts[i]; j++) {
+            Article article = new Article(cities[i] + " " + (j+1),
```

Comment:
Could you run the Java linter (if you were able to add it) to format this block of code?